legacy-adapter/src/Rsbc.Dmf.LegacyAdapter/Controllers/CasesController.cs

```diff
@@ -665,16 +665,95 @@ public ActionResult GetDocuments([FromRoute] string caseId)
                                 Driver = driver,
                                 ValidationMethod = validationMethod ?? string.Empty,
                                 ValidationPrevious = validationPrevious ?? string.Empty,
-                                Priority = priority ?? string.Empty,
-                                Owner = assign ?? string.Empty,
-                                SubmittalStatus = "Uploaded",
+                                Priority = "Regular" ?? string.Empty,
+                                Owner = "Client Services" ?? string.Empty,
+                                SubmittalStatus = "Uploaded" ?? string.Empty,
 
```

Comment:
there is no need to do the null check on a string constant... it's not wrong code but it also does not do anything.